bitnami/aspnet-core/templates/deployment.yaml

```diff
@@ -0,0 +1,170 @@
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
```

Comment:
If you have time, there are other values that we are including

- commonLabels
- commonAnnotations
- lifecycleHooks

Reply:
Example: harbor